IPython/utils/py3compat.py

```diff
@@ -1,22 +1,17 @@
 # coding: utf-8
```

Comment:
_if_ we remove the compatibility layer python2/python3, should this file still exists ? The way I see it either this file is / or is not. If there are function with use often they probably should be in utils. @takluyver thoughts ?

Reply:
Thoughts: leave it in for now, except for the if/else branch that only runs on Py2. Add a deprecation note in the docstring. Then, over a succession of future releases:

1. Eliminate all remaining usage of it in IPython
2. Add a deprecation warning when it's loaded
3. (Eventually) get rid of it

It doesn't cost much to leave it sitting around unused for a while, and other code might be importing it. We did [document it](http://ipython.readthedocs.io/en/4.x/development/pycompat.html) in earlier versions, though the [current version](http://ipython.readthedocs.io/en/stable/development/pycompat.html) tells people not to rely on it.